nbs/API/migrate.ipynb

```diff
@@ -1,23 +1,22 @@
 {
```

Comment:
<p>This was just moved from <code>processors.py</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>